common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -206,3 +206,36 @@ class VideoFields(object):
         scope=Scope.preferences,
         default=False,
     )
+
```

Comment:
why we need a blank line here? Please remove all blank lines between field definitions.